examples/simple_module/run.sh

```diff
@@ -5,9 +5,37 @@ cd `dirname $0`
 VENV_NAME="venv"
 PYTHON="$VENV_NAME/bin/python"
 
-python3 -m venv $VENV_NAME
-$PYTHON -m pip install -r requirements.txt -U # remove -U if viam-sdk should not be upgraded whenever possible
+if ! python3 -m venv $VENV_NAME >/dev/null 2>&1; then
+    echo "Failed to create virtualenv."
+    if command -v apt-get >/dev/null; then
+        echo "Detected Debian/Ubuntu, attempting to install python3-venv automatically."
+        SUDO="sudo"
+        if ! command -v $SUDO >/dev/null; then
+            SUDO=""
+        fi
+		if ! apt info python3-venv >/dev/null 2>&1; then
+			echo "package info not found, trying apt update"
+			$SUDO apt -qq update >/dev/null
+		fi
+        $SUDO apt install -qqy python3-venv >/dev/null 2>&1
+        if ! python3 -m venv $VENV_NAME >/dev/null 2>&1; then
+            echo "Failed to start module. This module requires Python >=3.8 and pip to be installed." >&2
+            exit 1
+        fi
+    else
+        echo "Failed to start module. This module requires Python >=3.8 and pip to be installed." >&2
```

Comment:
Re this error message -- could also get here if python is present but virtualenv is not.

(But not a merge blocker. This script may not need to be comprehensive; guessing the setup logic will live in its own tool)

Reply:
updated the error message